guides/whitelist.htm

```diff
@@ -32,7 +31,7 @@ <h5>Please note:</h5>
                 <div class='col-12 text-center'>
                     <h3>Greylist</h3>
                     <p>When projects are running out of work or are offline for temporary maintenance they are placed on the Greylist.  These projects will not reward Gridcoin until they meet the requirements to be placed back on the whitelist again.  Please see various social media (Twitter, Facebook, Reddit, Slack) for notices about greylisted projects.</p>
-                    <p>Please see <a href="https://gridcoin.ddns.net/pages/project-list.php">this unofficial Gridcoin Greylist monitoring website</a> for daily updates on which projects are added and removed from the Greylist.</p>
+                    <p>Please see <a href="https://gridcoin.ddns.net/pages/project-list.php">this unofficial Gridcoin Greylist monitoring website</a> for daily updates on which projects are added and removed from the Greylist. </p>
```

Comment:
Honestly, this line should probably be removed entirely. The greylist site by G_UK may never return

Reply:
Removed.